tests/sai_qualify/sai_infra.py

```diff
@@ -179,26 +179,31 @@ def run_case_from_ptf(duthost, dut_ip, ptfhost, test_case, test_interface_params
     logger.info("Running test: {0}".format(test_case))
     logger.info("Sleep {} sec between tests.".format(TEST_INTERVAL_IN_SEC))
     time.sleep(TEST_INTERVAL_IN_SEC)
-    cmds = []
     test_para = None
```

Comment:
Because `test_para` will be used in text formatting, it is better to assign default value `''` to `test_para`:

It's because `'{}'.format(None)` will be rendered to `'None'`.